charts/consul/values.yaml

```diff
@@ -230,7 +234,6 @@ global:
         #   {
         #     "connect": [{
         #       "ca_config": [{
-        #            "namespace": "my-vault-ns",
```

Comment:
@zalimeni Did you intent to remove this - to avoid confusion for operators?

Reply:
Yep! This seems no longer necessary to provide in the example because we now have an explicit field for it.